docs/instructions.md

```diff
@@ -54,5 +69,5 @@ Bitwarden on the Embassy is a *self-hosted* password manager, which means your p
 This portal enables you to configure environment settings, edit users, edit organizations and check version information. To use the admin portal: 
 
 1. Copy the admin token from `Properties`
-1. Navigate to your Bitwarden Tor address + `/admin` ie. `http://<your_bitwarden_tor_address>.onion/admin`
-1. Paste the admin token into the input field to login to the admin dashboard. 
\ No newline at end of file
+2. Navigate to your Bitwarden Tor address + `/admin` ie. `http://<your_bitwarden_tor_address>.onion/admin`
+3. Paste the admin token into the input field to login to the admin dashboard. 
```

Comment:
Markdown rendering should take care of this automatically. No need for changing these two lines.